### ISSUE TITLE
treat `import {} from "/foo"` as importing relative to the directory hsm is ran from

### ISSUE THREAD

**Issue by samualtnorman** [samualtnorman/hackmud-script-manager#166]
this will let people have setups that look like
```
src/
  my-user/
    local_script.ts
  global_script.ts
utils/
  shared-module.ts
package.json
README.md
```

**Comment by samualtnorman** [comment]
I think I'm chaning my mind on this, maybe `/` should be the target directory so HSE can have a layout that looks like this:
```
src/
  scripts/
    my_user/
      example_a.ts
    example_b.ts
  lib/
    isScriptor.ts
 ```
 where `example_a.ts` and `example_b.ts` can both do `import {} from "/lib/isScriptor"`

**Comment by samualtnorman** [comment]
supersedes #165

**Comment by samualtnorman** [comment]
okay so a complication of this is that it can't just be the target directory.
`import {} from "/lib/isScriptor"` in the above example would actually point at `src/scripts/lib/isScriptor` if this was the case which is not desirable

**Comment by samualtnorman** [comment]
fully implemented as of 2ed0c4fd5793d364eef6d5364971123fb12037d6
